### PR TITLE
Add test-datasets for cobra-meta nf-core module

### PR DESCRIPTION
**Pull request by CarsonJM**

### CODE REVIEW

File: data/genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/contig_id.tsv
Comment (sofstam) [inline data/genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/contig_id.tsv:1]: Should this be under a folder named `tsv` for example?

**Review by sofstam — COMMENTED** [review]
Nice! One comment and I assume this [file](https://github.com/nf-core/modules/blob/master/tests/config/test_data.config) needs to be updated as well.

File: data/genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/contig_id.tsv
Comment (CarsonJM) [reply data/genomics/prokaryotes/bacteroides_fragilis/illumina/fasta/contig_id.tsv:1]: That seems like a good idea to me 👍

**Review by sofstam — APPROVED** [review]
Nice!

**Pull request merged**